doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -31,9 +57,9 @@ The FIWARE NGSI (Next Generation Service Interface) API defines
 (whether to separate the two interfaces is currently under discussion).
 * a set of typical **roles** played by NGSI-compliant components
 
-### Context data modelling and exchange
+## Terminology
```

Comment:
From my point of view it is ok to have it flattened like suggested by @fgalan. 
I have not been following the formatting discussion too much, but in general I can support the approach to avoid having a too deep subsections tree.